src/js/tech/html5.js

```diff
@@ -275,14 +203,11 @@ class Html5 extends Tech {
 
     if ('webkitDisplayingFullscreen' in video) {
       this.one('webkitbeginfullscreen', function() {
-        this.player_.isFullscreen(true);
-
         this.one('webkitendfullscreen', function() {
-          this.player_.isFullscreen(false);
-          this.player_.trigger('fullscreenchange');
+          this.trigger('fullscreenchange');
```

Comment:
Ah, I see why the techFullscreenChange listeners were kept. These events won't really work though unless we're also setting isFullscreen to true or false somewhere also.